about/static/about.css

```diff
@@ -112,3 +114,29 @@ a.personal-link span.overlay {
 a.personal-link span.overlay:hover {
     box-shadow: 0px 0px 5px gray;
 }
+
+@media (max-width: 991px) {
+  div.member-info {
```

Comment:
Hmmm, wasn't working for me. Anyways, this was a 'i know it will for sure work though not the best practice' type of change, we're applying to sponsor stuff on Wednesday night, so we'll have to live with it in the short term.